xrdp/xrdp_mm.c

```diff
@@ -2303,7 +2288,7 @@ xrdp_mm_frame_ack(struct xrdp_mm *self, int frame_id)
         return 1;
     }
     ex = self->wm->client_info->max_unacknowledged_frame_count;
-    if (self->encoder->frame_id_client + ex > self->encoder->frame_id_server)
+    if (self->encoder->frame_id_client + ex >= self->encoder->frame_id_server)
```

Comment:
That change is needed in the case when max_unacknowledged_frame_count is 0. that happens with Parallels Client.
In that case the client would send a frame ack on frame 1, where the current server frame is also 1.